common/src/main/java/org/conscrypt/Conscrypt.java

```diff
@@ -67,6 +67,34 @@ public static X509TrustManager getDefaultX509TrustManager() throws KeyManagement
     }
 
     /**
+     * Utility that exposes common TLS constants.
+     */
+    public static final class Constants {
+        private Constants() {}
+
+        /**
+         * Returns the maximum length (in bytes) of plaintext data.
+         */
+        public static int maxPlaintextLength() {
```

Comment:
The JIT will make them constants at runtime. I'm not sure it matters a whole lot either way and methods are more consistent with the rest of this API. I also suspect that there are cases where methods provide more flexibility WRT API stability, although none come to mind ATM.